lib/config.ts

```diff
@@ -32,7 +32,7 @@ export class Config {
 	/**
 	 * Initialize getting the broker connection
 	 */
-	public async init(): Promise<void> {
+	public static async init(): Promise<void> {
```

Comment:
Although it is not in the diff, we shouldn't get the configs through params and through ENV Vars, also it shouldn't have default values, it should break if any var is missing.